packages/paste-website/src/pages/components/example-text/index.mdx

```diff
@@ -0,0 +1,112 @@
+import { ExampleText } from '@twilio-paste/example-text';
+import packageJson from '@twilio-paste/example-text/package.json';
+import {CodeBlock} from '@twilio-paste/code-block'
+import {InlineCode} from '@twilio-paste/inline-code'
+import {Label} from '@twilio-paste/label'
+import {Input} from '@twilio-paste/input'
+import {HelpText} from '@twilio-paste/help-text'
+import {Callout, CalloutHeading, CalloutText} from '@twilio-paste/callout'
+import {Paragraph} from '@twilio-paste/paragraph'
+
```

Comment:
You are missing the blockquote import so the example in the comparison table is just text and not the component

```suggestion
import { Blockquote, BlockquoteContent, BlockquoteCitation } from "@twilio-paste/blockquote";

```

Reply:
nice catch!